chef/automate
# Adding pagination to the scan history control

### :clipboard: Todo
- [x] Waiting on https://github.com/chef/automate/pull/4424 to merge to pass buildkite

### :nut_and_bolt: Description: What code changed, and why?

Adding pagination to the scan history to allow a user to see more than the first 10 scans. 

### :+1: Definition of Done

A user is able to see more than the first 10 scans on the compliance node page. 

### :athletic_shoe: How to Build and Test the Change
1. Start the automate-ui how you want. `build components/automate-ui`
1. Add some compliance nodes with ` chef_load_compliance_nodes 100`
1. Go to the compliance page "https://a2-dev.test/compliance/reports/overview"
1. Select one of the nodes tab 
![image](https://user-images.githubusercontent.com/1679247/97509708-44ee4780-1940-11eb-8fb7-2bce1a2b32f5.png)
1. Then select a node. 
1. Select the Scan History button 
![image](https://user-images.githubusercontent.com/1679247/97509761-66e7ca00-1940-11eb-9bdf-adedff778c5e.png)
1. Ensure there is a pagination control on the bottom and that it allows scrolling through the pages. 


### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
![image](https://user-images.githubusercontent.com/1679247/97502191-e28d4b00-192f-11eb-92b7-2fa90ae802ba.png)
